### ISSUE TITLE
AssertionError during Installation

### ISSUE THREAD

**Issue by dmtucker** [henry0312/pytest-docstyle#20]
Same issue as https://github.com/henry0312/pytest-codestyle/issues/21

```
$ pipenv run pip install pytest-pydocstyle
Collecting pytest-pydocstyle
Collecting pytest<4,>=3.0 (from pytest-pydocstyle)
  Using cached pytest-3.5.0-py2.py3-none-any.whl
Collecting pydocstyle<2.2,>=2.1 (from pytest-pydocstyle)
  Using cached pydocstyle-2.1.1-py3-none-any.whl
Requirement already satisfied: setuptools in /home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages (from pytest<4,>=3.0->pytest-pydocstyle)
Collecting attrs>=17.4.0 (from pytest<4,>=3.0->pytest-pydocstyle)
  Using cached attrs-17.4.0-py2.py3-none-any.whl
Requirement already satisfied: pluggy<0.7,>=0.5 in /home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages (from pytest<4,>=3.0->pytest-pydocstyle)
Requirement already satisfied: six>=1.10.0 in /home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages (from pytest<4,>=3.0->pytest-pydocstyle)
Collecting more-itertools>=4.0.0 (from pytest<4,>=3.0->pytest-pydocstyle)
  Using cached more_itertools-4.1.0-py3-none-any.whl
Requirement already satisfied: py>=1.5.0 in /home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages (from pytest<4,>=3.0->pytest-pydocstyle)
Collecting snowballstemmer (from pydocstyle<2.2,>=2.1->pytest-pydocstyle)
  Using cached snowballstemmer-1.2.1-py2.py3-none-any.whl
Installing collected packages: attrs, more-itertools, pytest, snowballstemmer, pydocstyle, pytest-pydocstyle
Exception:
Traceback (most recent call last):
  File "/home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages/pip/commands/install.py", line 342, in run
    prefix=options.prefix_path,
  File "/home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages/pip/req/req_set.py", line 784, in install
    **kwargs
  File "/home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages/pip/req/req_install.py", line 851, in install
    self.move_wheel_files(self.source_dir, root=root, prefix=prefix)
  File "/home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages/pip/req/req_install.py", line 1064, in move_wheel_files
    isolated=self.isolated,
  File "/home/david/.local/share/virtualenvs/backlog-eaCn-lpv/lib/python3.6/site-packages/pip/wheel.py", line 347, in move_wheel_files
    assert info_dir, "%s .dist-info directory not found" % req
AssertionError: pytest-pydocstyle .dist-info directory not found
```

**Comment by henry0312** [comment]
Please try the below and tell me the result.

```
pip install --no-cache --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pytest-pydocstyle
```

**Comment by dmtucker** [comment]
That seems to work:
```
(tmp-XVr6zr33) dtucker@dtucker-wkstn:/tmp $ pip install --no-cache --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pytest-pydocstyle
Collecting pytest-pydocstyle
  Downloading https://test-files.pythonhosted.org/packages/f7/f1/b85c3a5f616f03c34c966d869005d708463c7cf55e54109605b4cd31552b/pytest-pydocstyle-1.3.0.tar.gz
Collecting pytest<4,>=3.0 (from pytest-pydocstyle)
  Downloading https://files.pythonhosted.org/packages/ed/96/271c93f75212c06e2a7ec3e2fa8a9c90acee0a4838dc05bf379ea09aae31/pytest-3.5.0-py2.py3-none-any.whl (194kB)
    100% |████████████████████████████████| 194kB 3.9MB/s 
Collecting pydocstyle<2.2,>=2.1 (from pytest-pydocstyle)
  Downloading https://files.pythonhosted.org/packages/43/8a/f1141af0c8406788a5c38ad5001d4163d903b5384a6517239d2ac42734c8/pydocstyle-2.1.1-py3-none-any.whl
Collecting py>=1.5.0 (from pytest<4,>=3.0->pytest-pydocstyle)
  Downloading https://files.pythonhosted.org/packages/67/a5/f77982214dd4c8fd104b066f249adea2c49e25e8703d284382eb5e9ab35a/py-1.5.3-py2.py3-none-any.whl (84kB)
    100% |████████████████████████████████| 92kB 9.8MB/s 
Collecting attrs>=17.4.0 (from pytest<4,>=3.0->pytest-pydocstyle)
  Downloading https://files.pythonhosted.org/packages/b5/60/4e178c1e790fd60f1229a9b3cb2f8bc2f4cc6ff2c8838054c142c70b5adc/attrs-17.4.0-py2.py3-none-any.whl
Collecting six>=1.10.0 (from pytest<4,>=3.0->pytest-pydocstyle)
  Downloading https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Requirement already satisfied: setuptools in /home/dtucker/.local/share/virtualenvs/tmp-XVr6zr33/lib/python3.5/site-packages (from pytest<4,>=3.0->pytest-pydocstyle)
Collecting pluggy<0.7,>=0.5 (from pytest<4,>=3.0->pytest-pydocstyle)
  Downloading https://files.pythonhosted.org/packages/11/bf/cbeb8cdfaffa9f2ea154a30ae31a9d04a1209312e2919138b4171a1f8199/pluggy-0.6.0.tar.gz
Collecting more-itertools>=4.0.0 (from pytest<4,>=3.0->pytest-pydocstyle)
  Downloading https://files.pythonhosted.org/packages/7a/46/886917c6a4ce49dd3fff250c01c5abac5390d57992751384fe61befc4877/more_itertools-4.1.0-py3-none-any.whl (47kB)
    100% |████████████████████████████████| 51kB 20.1MB/s 
Collecting snowballstemmer (from pydocstyle<2.2,>=2.1->pytest-pydocstyle)
  Downloading https://files.pythonhosted.org/packages/d4/6c/8a935e2c7b54a37714656d753e4187ee0631988184ed50c0cf6476858566/snowballstemmer-1.2.1-py2.py3-none-any.whl (64kB)
    100% |████████████████████████████████| 71kB 32.8MB/s 
Installing collected packages: py, attrs, six, pluggy, more-itertools, pytest, snowballstemmer, pydocstyle, pytest-pydocstyle
  Running setup.py install for pluggy ... done
  Running setup.py install for pytest-pydocstyle ... done
Successfully installed attrs-17.4.0 more-itertools-4.1.0 pluggy-0.6.0 py-1.5.3 pydocstyle-2.1.1 pytest-3.5.0 pytest-pydocstyle-1.3.0 six-1.11.0 snowballstemmer-1.2.1
```

**Comment by henry0312** [comment]
Is this also ok with pipenv?

**Comment by dmtucker** [comment]
That was in a `pipenv shell`.

**Comment by dmtucker** [comment]
My issue seems to be resolved after clearing `pip`'s and `pipenv`'s cache and re-creating my virtualenvs:
``` sh
rm -rf ~/.cache/pip*
```

**Comment by dmtucker** [comment]
Huh... well maybe I spoke too soon. It has re-occurred after I changed my testenv deps in Tox.

**Comment by dmtucker** [comment]
```
Collecting pytest-pydocstyle==1.3.1
  Downloading https://test-files.pythonhosted.org/packages/cc/ce/5e93a169a1eb4f397fcab1894c83c1e4b8da1f1c0f8c3500b0f03255775a/pytest-pydocstyle-1.3.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-24rrxggc/pytest-pydocstyle/setup.py", line 5, in <module>
        version = open('VERSION').read().strip()
    FileNotFoundError: [Errno 2] No such file or directory: 'VERSION'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-24rrxggc/pytest-pydocstyle/
```
I think you need to add `VERSION` to [`MANIFEST.in`](https://packaging.python.org/tutorials/distributing-packages/#manifest-in).

**Comment by henry0312** [comment]
@dmtucker Thank you for your quick response.
I put MANIFEST.in and uploaded 1.3.1.dev1 on TestPyPI.
Can you re-try `pytest-docstyle==1.3.1.dev1` and `pytest-pydocstyle==1.3.1.dev1`?

**Comment by dmtucker** [comment]
I can't get either of those to break :sunglasses:

**Comment by henry0312** [comment]
Really? I think `1.3.1.dev1` is fixed.
I uploaded `1.3.1.dev5`, please try them.
Both ot them work for me.

```sh
# pytest-docstyle
pip install --no-cache --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pytest-docstyle==1.3.1.dev5

# pytest-pydocstyle
pip install --no-cache --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pytest-pydocstyle==1.3.1.dev5
```

**Comment by dmtucker** [comment]
I think 1.3.1.dev1 is fixed too. (Sorry, that's what I meant.)

**Comment by henry0312** [comment]
I see.
(I'm not good at English, sorry)

**Comment by henry0312** [comment]
I just released `v1.3.1` on PyPI (not TestPyPI).
It should be fixed, please try it.
